crates/red_knot_project/tests/check.rs

```diff
@@ -198,10 +198,10 @@ impl SourceOrderVisitor<'_> for PullTypesVisitor<'_> {
     fn visit_stmt(&mut self, stmt: &Stmt) {
         match stmt {
             Stmt::FunctionDef(function) => {
-                let _ty = function.ty(&self.model);
+                let _ty = function.get_type(&self.model);
```

Comment:
Getters are rather unidiomatic in the rust ecosystem. I think I prefer keeping `ty` here (`r#type` is too annoying to type). 

Or possibly `infer_type`?